src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -17,34 +15,67 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
-		return dao.findOne(id);
-
+		Eval eval = dao.findOne(id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "eval". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1488)